docs/config.md

```diff
@@ -230,6 +231,9 @@ names:
 * *loadBalancer* -- Optional.  Specifies a load balancer to use.  It must be an
 object containing keys:
   * *kind* -- One of the supported load balancers.
+  * *probationEnabled* -- Controls whether endpoints are eagerly evicted from
```

Comment:
Yeah, I totally agree with you re: boolean and that name... right now I'm still leaning towards matching finagle, lmk if you have a stronger opinion :)